pup-0004.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Code of Conduct
+author: Dennis Kliban
+created: 06-Dec-2017
+status: Active
```

Comment:
s/Active/Approved/

If this gets merged, it's approved :)

Reply:
+1 to changing this now so it's in it's final form.